### ISSUE TITLE
Deploy issue with mercury-rails (0.9.0)

### ISSUE THREAD

**Issue by vbklv** [sharetribe/sharetribe#1808]
##### Sharetribe Version:

176a4e7

##### Environment:

Engineyard on AWS
ruby 2.2.3p173 (2015-08-18 revision 51636) [x86_64-linux]

##### Server mode:

Staging

##### Description:

On deployment on the **bundle install** step I get this error:

SystemStackError: stack level too deep
An error occurred while installing mercury-rails (0.9.0), and Bundler cannot
continue.
Make sure that `gem install mercury-rails -v '0.9.0'` succeeds before bundling.

On the local OSX dev installation with Ruby 2.2.3 it works fine.

##### Steps To Reproduce:

Deployment on Engineyard staging environment

##### Additional information

N/A

**Comment by vbklv** [comment]
After updating **gem** from 2.4.8 (latest supported version by Engineyard) to 2.6.1, I get:

NoMethodError: undefined method `spec' for nil:NilClass
An error occurred while installing mercury-rails (0.9.0), and Bundler cannot
continue.
Make sure that `gem install mercury-rails -v '0.9.0'` succeeds before bundling.

**Comment by vbklv** [comment]
Issue has been resolved.

**Comment by rap1ds** [comment]
@vbklv Nice to hear that this has been resolved!

Could you summarize how you resolved it, so that if other people face the same issue they know how to fix it?

**Comment by vbklv** [comment]
Sure, I am not 100% sure what happened, but this is a short summary of what I have played with:
• I removed the `/data/APP_NAME/shared/bundled_gems` folder
• Then tried several times upgrading and downgrading `rubygems` (to latest supported v2.4.8)
• Then at some point mercury-rails installation worked with `gem install mercury-rails -v '0.9.0'`
• Then the deployment worked (with rubygems 2.4.8 and ruby 2.2.3)

**Comment by rap1ds** [comment]
@vbklv Thanks for the summary! :+1:
